Added Matrix_Sort.py

rohangawhade opened the pull request:
<!--
Thank you for sending the PR! We appreciate you spending the time to work on these changes.

Help us understand your motivation by explaining why you decided to make this change.
You can learn more about contributing to NeoAlgo here: https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md

Happy Contributing!

-->

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes
### Description

Added Matrix_Sort.py file which sorts the content of the matrix while keeping the structure of matrix unchanged.
### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#825

rohangawhade committed:
Added Matrix_Sort.py

iamrajiv reviewed (changes requested):
@rohangawhade Update the code according to PEP 8 rules, check your code here http://pep8online.com/. Add time and space complexity.

rohangawhade committed:
Update Matrix_Sort.py

Updated the Matrix_Sort according to PEP8 and added time and space complexity

rohangawhade commented:
> @rohangawhade Update the code according to PEP 8 rules, check your code here http://pep8online.com/. Add time and space complexity.

Updated the code according to the PEP8 rules from the link shared. Also added time and space complexity at the end of the code. @iamrajiv

iamrajiv commented on Python/sort/Matrix_Sort.py:
```diff
@@ -0,0 +1,57 @@
+# 7 8 9
+
+
+# time complexity = O(n^2)
```
@rohangawhade I think the time complexity will be `O(n^2log2(n))` since you have used `sort()` function.

rohangawhade replied:
Okay. So I'll update the time complexity. @iamrajiv

rohangawhade committed:
Update Matrix_Sort.py

Updated the time complexity from O(n^2) to O(n^2log2(n))

atarax665 commented on Python/sort/Matrix_Sort.py:
```diff
@@ -0,0 +1,57 @@
+def sort_matrix(mat, size):
```
Add docstring to explain how the function works.

rohangawhade replied:
Added the docstring and checked for PEP8 requirements too. @atarax665

rohangawhade committed:
Update Matrix_Sort.py

Added Docstrings to sort_matrix() and display_matrix() and checked for PEP8 requirements.

atarax665 reviewed (approved):
LGTM :)

iamrajiv reviewed (changes requested):
@rohangawhade Resolve the merge conflicts.

rohangawhade committed:
Merge branch 'master' into MatrixSort

rohangawhade commented:
> @rohangawhade Resolve the merge conflicts.

Done. @iamrajiv

HarshCasper commented:
@all-contributors bot please add @rohangawhade for code
